lms/djangoapps/instructor/views/api.py

```diff
@@ -1196,6 +1218,13 @@ def generate_registration_codes(request, course_id):
         internal_reference=internal_reference,
         customer_reference_number=customer_reference_number
     )
+    unit_price = (Decimal(sale_price)/course_code_number).quantize(Decimal('.01'), rounding=ROUND_DOWN)
+    invoice_item = CourseRegistrationCodeInvoiceItem.objects.create(
+        invoice=sale_invoice,
+        qty=course_code_number,
+        unit_price=unit_price,
```

Comment:
`unit_price` should be the *actual* price of the enrollment (determined from the course modes table).
`billed_unit_price` should be the price per enrollment specified by the sales rep.